src/main/java/com/conveyal/gtfs/util/json/JsonManager.java

```diff
@@ -0,0 +1,104 @@
+package com.conveyal.gtfs.util.json;
```

Comment:
This is probably an overcomplicated way to handle JSON serialization but it's fine for now.